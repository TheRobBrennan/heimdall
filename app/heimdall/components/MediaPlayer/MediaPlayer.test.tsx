import React from "react"
import TestRenderer from "react-test-renderer"

import Component from "./MediaPlayer"

describe("The MediaPlayer component", () => {
  it(`should render`, () => {
    // Verify success state
    const component = TestRenderer.create(<Component />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
