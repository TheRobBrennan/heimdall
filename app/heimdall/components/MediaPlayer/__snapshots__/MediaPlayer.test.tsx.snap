// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The MediaPlayer component should render 1`] = `
<div
  className="player-container"
>
  <div
    id="player"
  >
    <video
      controls={true}
      crossorigin={true}
      playsinline={true}
      poster="/__demo__/View_From_A_Blue_Moon_Trailer-HD.jpg"
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <source
        size="576"
        src="/__demo__/View_From_A_Blue_Moon_Trailer-576p.mp4"
        type="video/mp4"
      />
      <track
        default={true}
        kind="captions"
        label="English"
        src="/__demo__/captions/View_From_A_Blue_Moon_Trailer-HD.en.vtt"
        srcLang="en"
      />
      <track
        kind="captions"
        label="FranÃ§ais"
        src="/__demo__/captions/View_From_A_Blue_Moon_Trailer-HD.fr.vtt"
        srcLang="fr"
      />
      <a
        download={true}
        href="/__demo__/View_From_A_Blue_Moon_Trailer-576p.mp4"
      >
        Download
      </a>
    </video>
  </div>
</div>
`;
